openfpga_flow/misc/ys_tmpl_yosys_vpr_dsp_flow.ys

```diff
@@ -94,4 +94,4 @@ stat
 # Output netlists
 #########################
 opt_clean -purge
-write_blif ${OUTPUT_BLIF}
+write_blif -param ${OUTPUT_BLIF}
```

Comment:
Some test cases for FPGA-Verilog uses this script and just consider ``.blif`` as circuit format. To avoid any conflicts in future, would you mind create dedicated synthesis script for quicklogic devices?
My plan is to keep most regression tests as simple as it could be, while QuickLogic tests contain these advanced features.
Let me know what you think.